### PR TITLE
BloodHeartless suggestions

### PR DESCRIPTION
**Pull request by vtange**
small word choice changes

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -206,7 +206,7 @@ QUEST_LV_0200_20150317_000205	{memo X}If you are not in a hurry, please defeat t
 QUEST_LV_0200_20150317_000206	{memo X}Rebel Commander Ruklys, and the citizens here engaged in an evil conduct{nl}despite the selections from the great merchant, Premier Eminent. The result is this place.
 QUEST_LV_0200_20150317_000207	{memo X}That's not bad. Maybe the guards can use it.
 QUEST_LV_0200_20150317_000208	{memo X}After Medis Diena, the Stone Frosts are occuring irregularly.{nl}Due to irregular Stone Frosts, we lost many of our soldiers on this wasteland.{nl}And we set the petrification devices limitedly here at the square of lamentation and we check the frequency of the frosts and move out the soldiers accordingly.
-QUEST_LV_0200_20150317_000209	{memo X}Check the petrification devices if you want to bo loyal to the Kingdom and receive it's determination.
+QUEST_LV_0200_20150317_000209	{memo X}Check the petrification devices if you want to to loyal to the Kingdom and receive it's determination.
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv:209]: there's 2 'to's

to be\* ought to be what you are looking for.

Comment (terrapin47) [inline QUEST_LV_0200.tsv:209]: Mm, memo lines.

Comment (ttgmichael) [inline QUEST_LV_0200.tsv:209]: Lol! I didn't catch that.

doesn't matter then xD

Comment (BloodHeartless) [inline QUEST_LV_0200.tsv:209]: and "it's" too? / I wanted change too "a leader of a group" to "a group leader" =)

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Comment by BloodHeartless** [comment]
"to to" yet

**Pull request merged**